### ISSUE TITLE
Issue with boot

### ISSUE THREAD

**Issue by Greasye** [mrworf/photoframe#18]
Hi I'm really new to raspberry use. I found our picture frame project and it looked very easy for me to do. I have burned the image to my sd card and booted the pi zero W. I have let it run for upto 30 mins at the first boot i'm using a 32gb sd card. It powers on and my screen show that it's receiving data, but I'm not getting anything displayed on it. I can' proceed to the next step. Can you please help me?

**Comment by mrworf** [comment]
I'm unsure about what's going wrong. Do you see a message on the connected display to configure the frame or not? Also, did you edit the `wifi-config.txt` file which was on the SD card and add your wifi settings to it?

**Comment by Greasye** [comment]
Yes I changed the file with my wifi settings. The screen is on but it's not giving me any instructions.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by mrworf** [comment]
Not yet. But you can input other things such as landscape, colors etc.
